edge/pkg/edged/clcm/clcm_linux.go

```diff
@@ -0,0 +1,43 @@
+package clcm
```

Comment:
Why is this file named with "_linux.go"?

Reply:
I don't see anything special that needs to be build under linux. And it breaks build under other OS.

Reply:
Its based on the k8s kubelet...its using a linux directory path here

Reply:
No kubeletRootDir is just a const